### PR TITLE
Bumped versions, formatted and unified code

### PR DESCRIPTION
**Pull request by Keimeno**
I've noticed that in some parts of the code, single quotes and double quotes are mixed with each other, so I've set up a editorconfig, that is respected by formatters like prettier to only use single quotes in the files specified.

Also simplified some tag inclusions, like
```vue
<nest-footer></nest-footer>
```
to 
```vue
<nest-footer />
```

### CODE REVIEW

**Comment by MartonDev** [comment]
I'm not sure about the single quotes. I personally prefer them, but when @zorbyte addressed the codestyle of nest.land, he said we should use double quotes

**Comment by t8** [comment]
Thanks for the PR, @Keimeno! I'm personally a fan of double quotes as well.

**Comment by Keimeno** [comment]
Changed to double quotes now

File: src/components/auth/SignUp.vue [hunk 1]
```diff
@@ -26,117 +41,119 @@
...
+      if (this.password !== this.passwordConfirm) {
+        this.$emit("new-error", "Your passwords do not match.");
+        this.buttonStatusClass = "";
+        return;
+      }
+      if (this.username === this.password) {
+        this.$emit(
+          "new-error",
+          "You can't use the same username and password."
+        );
```
Comment (zorbyte) [inline src/components/auth/SignUp.vue:110]: Spacing between blocks would be great.

File: src/components/Footer.vue [hunk 2]
```diff
@@ -62,11 +57,14 @@
         <p class="subtitle is-6 has-text-white has-text-centered">
           nest.land is licensed under the
           <a href="http://opensource.org/licenses/mit-license.php">
-            <b>MIT License</b>
-          </a>.
+            <b>MIT License</b> </a
+          >.
```
[the remark this answers is not in the document]
Comment (Keimeno) [reply src/components/Footer.vue]: That seems to be a bug with prettier. Had to change those by hand.

File: src/components/auth/LogIn.vue [hunk 3]
```diff
@@ -20,89 +30,91 @@
...
+          Fetch your API Key
+        </button>
       </div>
     </div>
     <a
       class="has-text-dark has-text-underlined"
       @click="$emit('toggle-has-account', false)"
-    >No account? Sign up</a>
+      >No account? Sign up</a
+    >
```
[the remark this answers is not in the document]
Comment (Keimeno) [reply src/components/auth/LogIn.vue]: Related to https://github.com/nestdotland/nest.land/pull/233#discussion_r459148361

File: src/components/auth/SignUp.vue (hunk 1 above)
Comment (Keimeno) [reply src/components/auth/SignUp.vue]: Added now. But I think for the future it would be good separating those into multiple functions, or even mixins.

**Comment by Keimeno** [comment]
Resolved all requested changes and setup an eslintrc config file mentioning the comma dangle rule @zorbyte

**Comment by zorbyte** [comment]
My bad, github mobile did an oopsie.

**Comment by t8** [comment]
We're going to be merging one more PR into master before this. After that, this will be merged! Sorry for the delay, and thanks so much for the additions.

**Comment by t8** [comment]
Thanks so much for the contribution @Keimeno! Sorry it's taken so long to be merged. Feel free to email me if you'd like to join our internal chat!

tate@arceum.co

**Pull request merged**